components/hab/habitat/plan.sh

```diff
@@ -72,6 +73,10 @@ do_prepare() {
   # package proper--it won't find its way into the final binaries.
   export LD_LIBRARY_PATH=$(pkg_path_for gcc)/lib
   build_line "Setting LD_LIBRARY_PATH=$LD_LIBRARY_PATH"
+
+  # This doesn't need to have a value. Its presence alone is sufficient
+  # to trigger the behavior we want.
+  export JEMALLOC_SYS_DISABLE_STATIC
```

Comment:
Re-reading our `plan.sh`, I think in this case we actually want the static libraries, since we're setting `OPENSSL_STATIC=true`, `SODIUM_STATIC=true` and `LIBARCHIVE_STATIC=true`.  

I've also been unable to replicate the builds.  Reading through the jemalloc `configure.ac` and the `build.rs` for jemalloc-sys,  the variable would be `JEMALLOC_SYS_ENABLE_<STATIC|SHARED>=no`,  however jemalloc-sys only sets flags from those variables explicitly listed in the README so that never makes its way to the ./configure 😭

Reply:
Fear not, I have another idea for this.